modules/create_timepoint/test/create_timepointTest.php

```diff
@@ -70,33 +71,99 @@ function testCreateTimepointDoespageLoad()
      *
      * @return void
      */
-    function testCreateTimepointSelectSubproject()
+    function testCreateTimepoint()
+    {
+        $this->_createTimepoint('900000','Experimental','V1');
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("New time point successfully registered", $bodyText);
+
+    }
+    /**
+     * Tests that, create a timepoint and test the success link
+     * 
+     *
+     * @return void
+     */
+    function testCreateTimepointSuccessLink()
+    {
+        $this->_createTimepoint('900000','Experimental','V9');
+
+        $this->safeClick(WebDriverBy::LinkText("Click here to continue."));
+        sleep(3);
+        $bodyText = $this->webDriver->getPageSource();
+        $this->assertContains("900000", $bodyText);
```

Comment:
@driusan  I fixed it , $this->webDriver->findElement(WebDriverBy::cssSelector("body")) instead of $this->webDriver->getPageSource();